packages/client/src/containers/LandingPage/AddTeamMemberModal.jsx

```diff
@@ -14,26 +16,33 @@ export const AddTeamMemberModal = ({
     return null;
   }
   return (
-    <div>
-      <h2>Add Team Member</h2>
-      <input
-        type="text"
-        value={newMemberFirstName}
-        onChange={(e) => setNewMemberFirstName(e.target.value)}
-        placeholder="First Name"
-      />
-      <input
-        type="text"
-        value={newMemberLastName}
-        onChange={(e) => setNewMemberLastName(e.target.value)}
-        placeholder="Last Name"
-      />
-      <button type="button" onClick={handleAddMember}>
-        Add Member
-      </button>
-      <button type="button" onClick={() => setShowAddModal(false)}>
-        Cancel
-      </button>
+    <div className="container">
```

Comment:
I think `material-ui` should have a built in Modal already, which would make your job much much easier :D

Reply:
https://mui.com/material-ui/react-modal/